daemon/razer_daemon/daemon.py

```diff
@@ -223,9 +231,35 @@ def __init__(self, verbose=False, log_dir=None, console_log=False, run_dir=None,
         self.sync_effects(self._config.getboolean('Startup', 'sync_effects_enabled'))
         # TODO ======
 
-        # Setup quit signals
-        signal.signal(signal.SIGINT, self.quit)
-        signal.signal(signal.SIGTERM, self.quit)
+    def _init_signals(self):
+        # Heinous hack to properly handle signals on the mainloop
+        SIGS = [signal.SIGINT, signal.SIGTERM, signal.SIGHUP]
```

Comment:
If your declaring a constant, shouldnt you really stick it at the top, or not use an allcaps variable